### DISCUSSION TITLE
Can I do Redirect with status code 200 like Netlify?

### DISCUSSION THREAD

**Discussion by rafa-thayto** [vercel/community#706] [Help] [answered]
I need help to create a proxy using Vercel like **Netlify**, I've tried to use rewrites and redirects on the `vercel.json` file, and always when I've redirected, my URL changes. Can anyone help me with this or has had a similar problem?

My `vercel.json` file below
```json
{
  "redirects": [
    {
      "source": "/view-source",
      "destination": "https://github.com/vercel/vercel",
      "statusCode": 301
    },
    {
      "source": "/view-source2",
      "destination": "https://github.com/vercel/vercel",
      "statusCode": 200
    }
  ],
  "rewrites": [
    {
      "source": "/proxy/:match*",
      "destination": "https://thayto.com/:match*"
    }
  ]
}
```
I need that every time I go to `/proxy` for example my URL remains `myurl.com/proxy` and not `thayto.com/anything` like **Netlify** do on `_redirects` file
```
/proxy/* https://thayto.com/:splat
```

**Comment by amyegan** [comment, marked as answer]
Hi @rafa-thayto. Rewrites should do what you want https://vercel.com/docs/project-configuration#project-configuration/rewrites

**Reply by surjithctly** [reply]
I have a similar issue where `rewrites` not working as expected. See: https://github.com/vercel/vercel/discussions/8083
